site/build/handlebars-helpers/hyperlink.js

```diff
@@ -0,0 +1,19 @@
+
+function register(handlebars) {
+    handlebars.registerHelper('hyperlink', function(link, context) {
+        var pages = context.data.root.pages;
+        var matches = pages.filter(function(pageMetadata) {
+            return pageMetadata.page.destination.endsWith('/' + link);
+        });
+        if (matches.length === 1) {
+            var title = context.hash.title;
+            return '<a href=\"' + matches[0].page.destination + '\">' + title + '</a>';
+        } else {
+            throw new Error('Unable to locate link ' + link + ' within page ' + context.data.root.page.path);
```

Comment:
Possibly could throw different errors depending on whether `matches.length === 0` or `matches.length > 1`

In the latter case, something like:

```js
'Found too many (' + matches.length + ') matching links for ' + link + ' within page ' + context.data.root.page.path
```